src/mpl2/src/SimulatedAnnealingCore.h

```diff
@@ -126,15 +120,21 @@ class SimulatedAnnealingCore
 
   // Initialize the SA worker
   virtual void initialize() = 0;
-  // Run FastSA algorithm
+  virtual void run() = 0;
```

Comment:
Is fastSA the public access and run() for internal dispatch, or is there some case for clients to call run directly?  It is public.

Reply:
Actually it's the opposite. fastSA() should be private as it's called inside run(). I'll fix it